### PR TITLE
Bump to Node 10

### PR DESCRIPTION
**Pull request by jason-fox**
Node 8 leaves LTS at the end of the year. The default node version should be switched to Node Dubnium for the upcoming December FIWARE 7.8.1 Patch release.

### CODE REVIEW

**Comment by fgalan** [comment]
It seems there is a conflict in CHANGES_NEXT_RELEASE... as soon as it gets solved, I'll merge.

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- Use device apikey if exists in getEffectiveApiKey for command handling
+- Upgrade NodeJS version from 8.16.1 to 10.17.0 in Dockerfile due to Node 8 End-of-Life
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: I think some similar to https://github.com/telefonicaid/iotagent-ul/pull/398 has happened here.

Comment (jason-fox) [reply CHANGES_NEXT_RELEASE]: Fixed 7d0a33a

**Comment by fgalan** [comment]
imagen

This is funny... how a change in the Dockerfile (which is not used by travis/coveralls) can produce a change in the coverage metric? A mistery to me :)

**Pull request merged**